src/fparser/common/readfortran.py

```diff
@@ -1278,7 +1298,7 @@ def handle_multilines(self, line, startlineno, mlstr):
             suffix, qc, had_comment = self.handle_inline_comment(suffix, self.linecount)
             # no line continuation allowed in multiline suffix
             if qc is not None:
-                message = "following character continuation: {!r}," + " expected None."
+                message = "following character continuation: {!r}," " expected None."
```

Comment:
Not quite sure what's going on here. Could we just have a single string?

Reply:
Black only seems to introduce some whitespaces around `:`, e.g.:
```
-            s = ' '+s
-        tab = tab[len(s):]
+            s = " " + s
+        tab = tab[len(s) :]
```
which my pycodestyle/pylint complain about. But black is now happy ;)